web/src/CityContentSwitcher.tsx

```diff
@@ -94,21 +92,19 @@ const CityContentSwitcher = ({ cities, languageCode }: CityContentSwitcherProps)
   }
 
   const cityRouteProps: CityRouteProps = {
-    cities,
-    languages: cityModel.languages,
-    cityModel,
-    languageModel,
+    city,
     pathname,
     cityCode,
     languageCode,
   }
-  const { eventsEnabled, offersEnabled } = cityModel
-  const localNewsEnabled = buildConfig().featureFlags.newsStream && cityModel.localNewsEnabled
-  const tuNewsEnabled = buildConfig().featureFlags.newsStream && cityModel.tunewsEnabled
-  const poisEnabled = buildConfig().featureFlags.pois && cityModel.poisEnabled
+
+  const eventsEnabled = !city || city.eventsEnabled
+  const offersEnabled = !city || city.offersEnabled
+  const localNewsEnabled = buildConfig().featureFlags.newsStream && (!city || city.localNewsEnabled)
+  const tuNewsEnabled = buildConfig().featureFlags.newsStream && (!city || city.tunewsEnabled)
+  const poisEnabled = buildConfig().featureFlags.pois && (!city || city.poisEnabled)
```

Comment:
If the city is not available yet nothing is rendered in the routes anyway, therefore render the route until we know if its enabled.